local_pathfinding/ompl_path.py

```diff
@@ -34,6 +36,12 @@ def __init__(self, local_path_state: LocalPathState):
         self.start_state = (0.5, 0.4)
         self.goal_state = (0.5, -0.4)
 
+        self.reference_latlon = (
+            local_path_state.global_path[-1]
+            if local_path_state and len(local_path_state.global_path) > 0
+            else HelperLatLon(latitude=0.0, longitude=0.0)
+        )
```

Comment:
Now that I think about it more this is still not right. I'm thinking in LocalPathState we need to find the waypoint on the global path we're heading to (not always the last element), which will be our reference latlon here. Maybe this can be your next task Jay (doesn't have to be tackled in this PR, but let's at least write an issue for properly populating OMPLPathState).

Reply:
Issue #79 has been made from this issue.